core/src/main/java/discord4j/core/object/entity/User.java

```diff
@@ -85,7 +85,18 @@ public UserData getUserData() {
     }
 
     /**
-     * Gets the user's username, not unique across the platform.
+     * Gets the user's global username, not enforced to be unique across the platform.
+     * May be empty if the user has not set a global username.
+     *
+     * @return The user's global name
+     */
+    public final Optional<String> getGlobalName() {
+        return data.globalName();
+    }
```

Comment:
Since the transition might be shorter than initially expected, this should return `@Nullable` String.

This way we avoid breaking API later